specification/web/resource-manager/Microsoft.Web/stable/2023-12-01/CommonDefinitions.json

```diff
@@ -3649,6 +3653,41 @@
         }
       }
     },
+    "SiteDnsConfig": {
+      "type": "object",
+      "properties": {
+        "dnsServers": {
+          "description": "List of custom DNS servers to be used by an app for lookups. Maximum 5 dns servers can be set.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "dnsAltServer": {
+          "description": "Alternate DNS server to be used by apps. This property replicates the WEBSITE_DNS_ALT_SERVER app setting.",
+          "type": "string"
+        },
+        "dnsRetryAttemptTimeout": {
+          "format": "int32",
+          "description": "Timeout for a single dns lookup in seconds. Allowed range: 1-30. Default is 3.",
+          "type": "integer"
+        },
+        "dnsRetryAttemptCount": {
+          "format": "int32",
+          "description": "Total number of retries for dns lookup. Allowed range: 1-5. Default is 3.",
+          "type": "integer"
+        },
+        "dnsMaxCacheTimeout": {
+          "format": "int32",
+          "description": "Custom time for DNS to be cached in seconds. Allowed range: 0-60. Default is 30 seconds. 0 means caching disabled.",
+          "type": "integer"
+        },
+        "dnsLegacySortOrder": {
+          "description": "Indicates that sites using Virtual network custom DNS servers are still sorting the list of DNS servers. This is a view only property and can not be modified by a call against the site.",
+          "type": "boolean"
```

Comment:
consider using an enum instead   : [https://github.com/Azure/azure-openapi-validator/blob/main/docs/enum-instead-of-boolean.md](https://github.com/Azure/azure-openapi-validator/blob/main/docs/enum-instead-of-boolean.md)

Also please consider updating the name of the property to reflect the function.

Reply:
There is no possibility for this property to have more than 2 values in future so boolean should be fine here.